lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -24,7 +30,8 @@ def show_header?
       end
 
       def show_related_links?
-        step_navs.any? && (step_navs.count < 5 || active_step_by_step?)
+        return true if active_step_by_step?
+        step_navs.any? && step_navs.count < 5
```

Comment:
Did method need to change? How is the logic different?

Reply:
this will check `active_step_by_step?` which checks related and step navs. 

if we have it the way it was before, then we'd only check if there is any step navs and not check for related to step navs

Reply:
@DilwoarH We've talked about this in person, but at it's core the logic being used here hasn't changed. If you think it is worth the refactor though, can you separate it out from the rest of the changes? Thanks

Reply:
Can you split this code up a bit so that it's easier to follow, and add the comment above to the commit message. Cheers.